---
highlight_name: true
superuser: true
title: Da Kuang
first_name: Da
last_name: Kuang
role: Computer Science Doctoral Candidate
organizations:
- name: University of Pennsylvania
profiles:
- icon: brands/linkedin
  url: https://www.linkedin.com/in/kuang-da/
- icon: academicons/google-scholar
  url: https://scholar.google.com/citations?user=XsQIl3YAAAAJ&hl=en
- icon: brands/github
  url: https://github.com/kuang-da  
- icon: brands/x
  url: https://x.com/DaKuangDerek  
education:
- area: MA in Statistics
  institution: University of Pennsylvania, Wharton
- area: MSE in Computer and Information Science
  institution: University of Pennsylvania, Engineering
- area: MSE in Nanotechnology
  institution: University of Pennsylvania, Engineering 
- area: BSc in Electronic Science
  institution: Jilin University
interests:
- Representation & Metric Learning
- Embedding Design
- LLMs for Scientific Discovery
- Single-cell & Spatial Omics
---
## About Me

Hi! I am a final-year Ph.D. Candidate in the Department of Computer and Information Science at the University of Pennsylvania, advised by Professor [Junhyong Kim](https://www.bio.upenn.edu/people/junhyong-kim). I am funded by The Human BioMolecular Atlas Program ([HuBMAP](https://www.nature.com/articles/s41556-023-01194-w)) to develop computational frameworks for spatial biology.

My research centers on **representation learning**, particularly in designing robust and biologically meaningful embeddings to enable **automated scientific discovery**. I work at the intersection of **machine learning**, **single-cell genomics**, and **computational biology**, with ongoing efforts in:

- Learning Hierarchical representation: [CellTreeQM](https://arxiv.org/abs/2503.13925)
- LLM-based automatic scientific discovery: [Aegle](https://github.com/kimpenn/aegle) (ongoing)
- Robust representation: [BooleanSpurious](https://arxiv.org/abs/2403.03375), [RFNorm](https://openreview.net/forum?id=QLydjLdcFV&referrer=%5Bthe%20profile%20of%20Da%20Kuang%5D(%2Fprofile%3Fid%3D~Da_Kuang2))
- Design Protein Embedding Space by Empirical Kernel: [PESS](https://www.sciencedirect.com/science/article/abs/pii/S0022283622002789)
- Construct integrative embedding space for cellular developmental preturbation responses: [Argos](https://www.frontiersin.org/journals/drug-delivery/articles/10.3389/fddev.2024.1359700/full) (ongoing)


<!-- I am funded by The Human BioMolecular Atlas Program ([HuBMAP](https://www.nature.com/articles/s41556-023-01194-w)) to develop [computational frameworks](https://github.com/kimpenn/aegle) for analyzing spatial proteomics and transcriptomics data across organs and donors for Female Reproductive System. -->

<!-- Beyond my primary research, I have contributed to several projects in computational biology, including: [📐](https://openreview.net/forum?id=QLydjLdcFV) Developing a robust normalization method for single-cell RNA-seq data ; [🌀](https://www.sciencedirect.com/science/article/abs/pii/S0022283622002789) Learning from an empirical kernel space for efficiently predicting protein folding; [📈](https://www.frontiersin.org/journals/drug-delivery/articles/10.3389/fddev.2024.1359700/full) Time-series analysis of cellular phototransfection responses. I am also passionate about building data science tools to facilitate collaborative research. -->

I previously interned as a **Senior Data Scientist at IBM’s Chief Analytics Office**, where I worked on model deployment and decision optimization.

**I am actively seeking full-time opportunities as a Research Scientist in AI4Science across tech, biotech, or pharmaceutical industries. Feel free to reach out!**
